### ISSUE TITLE
sonar-scanner doesn't seem to scan javascript files with the *.mjs extension (esm modules)

### ISSUE THREAD

**Issue by ivanwills** [SonarSource/SonarJS#3120]
Javascript files with the mjs extension are needed by node to run as es6 modules. Having recently converted 1 repository to using it I found sonar no longer scans those files.

I'm using sonar-scanner version 4.0.0.1744 with SonarQube version 7.9

```
$ cat sonar-project.properties 
sonar.projectKey=slow-crawler
sonar.projectName=slow-crawler
sonar.projectVersion=1.0.0
false
sonar.sources=src,scripts
sonar.sourceEncoding=UTF-8
```

If I copy all the *.mjs to *.js sonar-scanner will find those files

**Comment by vilchik-elena** [comment]
Hi, first please consider updating your SQ to latest version or LTS (8.9.x).
You can configure the suffixes with `sonar.javascript.file.suffixes` (also available in UI)

**Comment by ivanwills** [comment]
If only I cold update the company SQ it's not in my control. Locally I've tried the Docker 8.9-comunity image (which is a lot more difficult and 7.9 to get started) and it look like its fixed in 8.9
